CMakeLists.txt

```diff
@@ -111,8 +111,8 @@ ENDIF()
 
 
 set(Kokkos_VERSION_MAJOR 3)
-set(Kokkos_VERSION_MINOR 1)
-set(Kokkos_VERSION_PATCH 1)
+set(Kokkos_VERSION_MINOR 2)
+set(Kokkos_VERSION_PATCH 0)
```

Comment:
I am honestly not sure what is the right thing to do on develop.  How we can signify that there is departure from the actual release.  Anyway I don't expect us to resolve that here.